### ISSUE TITLE
USBD_CDC_ACM_HandleTypeDef dubplicate data for NUMBER_OF_CDC?

### ISSUE THREAD

**Issue by devOramaMan** [alambe94/I-CUBE-USBD-Composite#20]
typedef struct
  {
    uint32_t data[**NUMBER_OF_CDC**][CDC_DATA_HS_MAX_PACKET_SIZE / 4U]; /* Force 32bits alignment */
    uint8_t CmdOpCode;
    uint8_t CmdLength;
    uint8_t *RxBuffer;
    uint8_t *TxBuffer;
    uint32_t RxLength;
    uint32_t TxLength;

    __IO uint32_t TxState;
    __IO uint32_t RxState;
  } USBD_CDC_ACM_HandleTypeDef;

and 

USBD_CDC_ACM_HandleTypeDef CDC_ACM_Class_Data[**NUMBER_OF_CDC**];

I couldn't quite understand from the code why data also has “NUMBER_OF_CDC” array aswell as the CDC_ACM_Class_Data. Is it something I’m missing?

**Comment by alambe94** [comment]
Yes, you are correct. [NUMBER_OF_CDC] is not necessary in USBD_CDC_ACM_HandleTypeDef.
